client/platform/desktop/frontend/components/Recent.vue

```diff
@@ -262,69 +335,67 @@ export default defineComponent({
-                    <v-icon>
-                      mdi-spin mdi-sync
-                    </v-icon>
-                  </span>
-                </span>
-                <router-link
-                  v-else
-                  :to="{ name: 'viewer', params: { id: recent.id } }"
-                  class="primary--text text--lighten-3 text-decoration-none"
-                >
-                  {{ recent.name }}
-                </router-link>
-                <span class="grey--text px-4">
-                  {{ recent.originalBasePath }}
+              </template>
+              <template #[`item.name`]="{ item }">
+                <span :key="item.id">
+                  <div v-if="setOrGetConversionJob(item.id)">
+                    <span class="primary--text text--darken-1 text-decoration-none">
+                      {{ item.name }}
+                    </span>
+                    <span class="pl-4">
+                      Converting
+                      <v-icon>
+                        mdi-spin mdi-sync
+                      </v-icon>
+                    </span>
+                  </div>
+                  <div
+                    v-else
+                    class="link primary--text text--lighten-3"
+                    @click="preloadCheck(item)"
+                  >
+                    {{ item.name }}
+                  </div>
+                  <div class="grey--text">
+                    {{ item.originalBasePath }}
+                  </div>
                 </span>
-              </h3>
-            </div>
-            <div
-              v-if="pageSize < totalRecents"
-              class="mx-1"
-            >
-              <v-divider class="my-2" />
-              <h3
-                class="text-body-1"
-                style="cursor: pointer;"
-                @click="toggleMore"
-              >
-                <v-icon
-                  class="pr-2"
-                  color="primary lighten-3"
-                >
-                  {{ (limit === totalRecents) ? 'mdi-chevron-up' : 'mdi-chevron-down' }}
-                </v-icon>
+              </template>
+              <template #[`item.accessedAt`]="{ item }">
                 <span
-                  class="primary--text text--lighten-3"
+                  :key="item.id"
+                  class="grey--text text-body-2"
                 >
-                  <span v-if="limit < totalRecents">
-                    Show {{ totalRecents - pageSize }} more
-                  </span>
-                  <span v-else>
-                    Show less
-                  </span>
+                  {{ toDisplayString(item.accessedAt) }}
                 </span>
-              </h3>
-            </div>
+              </template>
+              <template #[`item.delete`]="{ item }">
+                <v-btn
+                  :key="item.id"
+                  color="error"
+                  icon
+                  @click="confirmDeleteDataset(item.id, item.name)"
+                >
```

Comment:
If you do go with my suggestion on removing the `Delete` header from the delete button column, it might be worth adding a hover tooltip similar to the viewer:

![Screenshot from 2021-07-26 09-59-50](https://user-images.githubusercontent.com/30541973/127001797-bc007faf-b81f-43b0-9748-a43c8a28add0.png)

Again, just throwing the idea out there. Feel free to ignore if its too extra.